### PR TITLE
Adding symmetric key sample to provisioning device samples folder

### PR DESCRIPTION
**Pull request by timtay-microsoft**

### CODE REVIEW

**Comment by prmathur-microsoft** [comment]
@CIPop for review

**Comment by Behrad-iz** [comment]
Could we have this branch merged 
Thanks

**Comment by timtay-microsoft** [comment]
I'm still waiting on the C# SDK to release a nuget containing this feature. This sample will not compile until that next release, so I cannot merge this in yet.

File: provisioning/Samples/device/SymmetricKeySample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,123 @@
+﻿using Microsoft.Azure.Devices.Provisioning.Client;
```
Comment (abhipsaMisra) [inline provisioning/Samples/device/SymmetricKeySample/Program.cs]: Copyright and license statement missing

File: provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\Common\ProvisioningDeviceClientSample.cs" Link="ProvisioningDeviceClientSample.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.20.0" />
+    <PackageReference Include="Microsoft.Azure.Devices.Provisioning.Security.Tpm" Version="1.1.5" />
```
Comment (abhipsaMisra) [inline provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj]: Do we need tpm package dependency here?

**Comment by abhipsaMisra** [comment]
Thanks @timtay-microsoft !
Looks good; I would rather have the individual and group enrollments as separate samples though, since they have different use-cases.
Also, do we have service client samples for creating symmetric key enrollments?

File: provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj (hunk 2 above)
Comment (timtay-microsoft) [reply provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj]: We do not. Good catch

File: provisioning/Samples/device/SymmetricKeySample/Program.cs (hunk 1 above)
Comment (timtay-microsoft) [reply provisioning/Samples/device/SymmetricKeySample/Program.cs]: I'll add it, good catch

**Comment by timtay-microsoft** [comment]
I'm just following how the TPM and x509 samples were drawn up, so I'm keeping the group and individual samples together. We do not, and I can add those later if needed. I'm going to merge this in for now, though.

**Pull request merged**